<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userwindow</class>
 <widget class="QMainWindow" name="userwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>busbee</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;</string>
    </property>
    <property name="text">
     <string>[+] status</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>260</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(249,191,81);
font: 700 11pt &quot;Segoe UI&quot;;
color: black;
border-radius: 20px;
</string>
    </property>
    <property name="text">
     <string>Go Back</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>331</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="login">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(249,191,81);
font: 700 11pt &quot;Segoe UI&quot;;
color: black;
border-radius: 20px;
</string>
     </property>
     <property name="text">
      <string>Log In</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_username">
        <property name="styleSheet">
         <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;
color: black;</string>
        </property>
        <property name="text">
         <string>  Username  :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_username"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_password">
        <property name="styleSheet">
         <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;
color: black;</string>
        </property>
        <property name="text">
         <string>  Password   :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_password">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="login_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>180</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(249,191,81);
font: 700 09pt &quot;Segoe UI&quot;;
color: black;
border-radius: 20px;
</string>
     </property>
     <property name="text">
      <string>Create New Account</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>221</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resourse.qrc">:/img/images/userloginlogo.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resourse.qrc">:/img/images/BusBee_sgnin.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resourse.qrc"/>
 </resources>
 <connections/>
</ui>
